src/utils/deployContract.js

```diff
@@ -0,0 +1,86 @@
+import Web3 from "web3";
+import FestakingJson from "../contracts/Festaking.json";
+import FerrumJson from "../contracts/FerrumToken.json";
+import FestakingTest from "../contracts/FestakingTest.json";
+
+const deployContarct = async (type = "default", data = {}) => {
+  const web3 = new Web3("ws://localhost:7545");
```

Comment:
Rename this to deployContractForTest(), create a deployContract() for prod that only deploys the contract. Fund movement is not necessary in prod